Deployment/VMware vSphere/High Availability/variables.tf

```diff
@@ -229,24 +214,20 @@ variable "DSX-datadisk" {
 
 variable "vm-domain" {
   type        = string
-  description = "Linux virtual machine domain name for the machine"
-  default     = "<Domain name>"
+  description = "Domain name for the VMs."
 }
 
 variable "prefix" {
   type        = string
-  description = "Prefix for vm name and DSX hostname"
-  default = "<string prefix, i.e. tf"
+  description = "Prefix for vm name and DSX hostname. Example: first-last-setup01"
```

Comment:
Nit: the prefix applies to all hosts, so it should say as much. At the very least capitalise VM to stick with convention.